src/Mono.Android/metadata

```diff
@@ -1381,6 +1381,25 @@
   <attr path="/api/package[@name='android.media']/interface[@name='MediaCas.EventListener']" name="argsType">MediaCasEventArgs</attr>
   <attr path="/api/package[@name='android.media']/interface[@name='MediaCas.EventListener']/method[@name='onEvent']/parameter[@name='MediaCas']" name="managedName">mediaCas</attr>
 
+  <!-- Until API Level 26, android.graphics.Color was static. In API Level 26,
+  it became instantiable and got instance members.
+
+  Xamarin.Android had wrong premise that android.graphics.Color can be totally
+  different thing and thus introduced completely different API with the same
+  type name. Wrong wrong wrong. It was wrong decision made back in 2010.
+
+  We will have to enable this line and bring back some sanity.
+  -->
+  <!-- attr path="/api/package[@name='android.graphics']/class[@name='Color']" name="managedName">AndroidColor</attr -->
+
+  <!-- API Level 27 -->
+
+  <!-- android.graphics.Color started to be in actual use and that uncovered
+  Xamarin.Android design mistake above! We have to remove WallpaperColors API
+  until Xamarin.Android fixes their bogus API design. -->
```

Comment:
Uh...what? Adding `Android.Graphics.Color` as a struct may have been "wrong decision made back in 2010," but it's not one we can change until we break API compatibility, which isn't going to happen anytime soon.

As such, we need to find an alternative that doesn't involve API breakage. Is there something wrong with the commented-out `<attr/>` which renames `android.graphics.Color` to `Android.Graphics.AndroidColor`?

Even without that `<attr/>`, why can't [`android.app.WallpaperColors`](https://developer.android.com/reference/android/app/WallpaperColors.html) be bound? I don't see anything in there which would prevent it from being bound *at all*. (It might be *useless*, lacking the `getPrimaryColor()` and related methods, but the type itself could still *exist*.)

Reply:
As commented above, changing _Color to something like AndroidColor causes breakage that needs complicated generator fixes (compared to just bringing in API Level 27, and such "fixes" should not block "enable API Level 27" effort).